.golangci.json

```diff
@@ -3,11 +3,28 @@
 		"disable-all": true,
 		"enable": [
 			"govet",
-			"golint",
-			"goimports"
+			"revive",
+			"goimports",
+      "gofmt"
```

Comment:
How ironic.

Reply:
yay for per-file indentation conventions that my editor hides...